apps/api-extractor/src/DebugRun.ts

```diff
@@ -2,30 +2,61 @@
 // See LICENSE in the project root for license information.
 
 // NOTE: THIS SOURCE FILE IS FOR DEBUGGING PURPOSES ONLY.
-//       IT IS INVOKED BY THE "Run.cmd" AND "Debug.cmd" BATCH FILES.
+//       IT IS INVOKED BY THE 'Run.cmd' AND 'Debug.cmd' BATCH FILES.
 
 import { Extractor } from './extractor/Extractor';
+import * as path from 'path';
 
 const extractor: Extractor = new Extractor(
   {
     compiler: {
       configType: 'tsconfig',
-      rootFolder: '.'
+      overrideTsconfig: {
+        'compilerOptions': {
+          'target': 'es6',
+          'forceConsistentCasingInFileNames': true,
+          'module': 'commonjs',
+          'declaration': true,
+          'sourceMap': true,
+          'experimentalDecorators': true,
+          'types': [
+            'node'
+          ],
+          'lib': [
+            'es5',
+            'scripthost',
+            'es2015.collection',
+            'es2015.promise',
+            'es2015.iterable',
+            'dom'
+          ],
+          'strictNullChecks': true
+        },
+        'include': [ 'lib/**/*.d.ts' ]
+      },
+      rootFolder: '../../libraries/node-core-library'
     },
     project: {
-      entryPointSourceFile: 'src/index.ts',
-      externalJsonFileFolders: ['./testInputs/external-api-json' ]
+      entryPointSourceFile: 'lib/index.d.ts',
+      externalJsonFileFolders: [ ]
     },
     apiReviewFile: {
-      enabled: true,
-      apiReviewFolder: __dirname
+      enabled: false,
+      apiReviewFolder: path.join(__dirname, 'debug')
     },
     apiJsonFile: {
-      enabled: true
+      enabled: true,
+      outputFolder: path.join(__dirname, 'debug')
+    },
+    packageTypings: {
+      enabled: true,
+      outputFolder: path.join(__dirname, 'debug')
     }
   }
 );
 
+console.log('CONFIG:' + JSON.stringify(extractor.config, undefined, '  '));
+
 extractor.analyzeProject();
```

Comment:
>analyzeProject [](start = 10, length = 14)

Rename to `processProject`

Reply:
Fix in a separate PR.

---
In reply to: [152421384](https://github.com/Microsoft/web-build-tools/pull/448#discussion_r152421384) [](ancestors = 152421384)